.circleci/config.yml

```diff
@@ -16,6 +16,16 @@ jobs:
           path: reports/junit
       - store_artifacts:
           path: reports/junit
+  test-smapi:
+    docker:
+      - image: circleci/node:8
+    environment:
+      SMAPI: true
+    steps:
+      - checkout
+      - run: npm install
+      - run: sudo npm install ask-cli -g
```

Comment:
I'm not sure what you mean here, but the tests have to use an environment variable, because the CI workflow cannot perform the login. 

For users using this locally, it is required that they initialize the ASK CLI. Either one will work. If they do that, the tool will correctly use the values from the config file the CLI creates.

Reply:
The user can use it, but i mean that in circle as it is a new environment everytime ask-cli won't be initialized correctly. And this is the configuration for the circle test specifically

Reply:
It works fine in circle because it takes the environment variable and creates a file based on that.